Framework/Test_Framework_UnitTest.rb

```diff
@@ -0,0 +1,240 @@
+    assert_equal("Diver Closed", create_testframe_instance.close_site())
+  end
+
+  def test_sleep_time
+    create_testframe_instance = SeleniumFrameWork.new()
+    sec = 5
+    assert_equal("Sleep Time Invoked", create_testframe_instance.sleep_time(sec))
+    create_testframe_instance.close_site()
+  end
+
+  def test_negative_sleep_time
+    create_testframe_instance = SeleniumFrameWork.new()
+    sec = 'a'
+    assert_equal("Sleep Time not Invoked", create_testframe_instance.sleep_time(sec))
+    create_testframe_instance.close_site()
+  end
+
+  def test_max_window
+    create_testframe_instance = SeleniumFrameWork.new()
+    assert_equal("Window Maximized", create_testframe_instance.maximize_window())
+    create_testframe_instance.close_site()
+  end
+
+  def test_get_title
+    create_testframe_instance = SeleniumFrameWork.new()
+    url = "file://src/Test_Framework.html"
+    create_testframe_instance.open_site(url)
+    assert_equal("Page Title: Ruby FrameWork Test", create_testframe_instance.get_title())
+    create_testframe_instance.close_site()
+  end
+
+  def test_get_negative_title
+    create_testframe_instance = SeleniumFrameWork.new()
+    url = "file://src/Test_Framework.html"
+    create_testframe_instance.open_site(url)
+    assert_not_equal("Page Title: Test", create_testframe_instance.get_title())
+    create_testframe_instance.close_site()
+  end
+
+  def test_find_element
+    create_testframe_instance = SeleniumFrameWork.new()
+    url = "file://src/Test_Framework.html"
+    create_testframe_instance.open_site(url)
+    assert_equal(false, create_testframe_instance.findelement("id", "name").nil?)
+    create_testframe_instance.close_site()
+  end
+
+  def test_find_invalid_element
+    create_testframe_instance = SeleniumFrameWork.new()
+    url = "file://src/Test_Framework.html"
+    create_testframe_instance.open_site(url)
+    assert_equal(nil, create_testframe_instance.findelement("id", "phone"))
+    create_testframe_instance.close_site()
+  end
+
+  def test_click_button
+    create_testframe_instance = SeleniumFrameWork.new()
+    url = "file://src/Test_Framework.html"
+    create_testframe_instance.open_site(url)
+    create_testframe_instance.scoll_by(1)
+    assert_equal("Button Clicked", create_testframe_instance.click_btn("id", "submit"))
+    create_testframe_instance.close_site()
+  end
+
+  def test_click_invalid_button
+    create_testframe_instance = SeleniumFrameWork.new()
+    url = "file://src/Test_Framework.html"
+    create_testframe_instance.open_site(url)
+    assert_equal("Button not Clicked", create_testframe_instance.click_btn("id", "clear"))
+    create_testframe_instance.close_site()
+  end
+
+
+  def test_send_keys
+    create_testframe_instance = SeleniumFrameWork.new()
+    url = "file://src/Test_Framework.html"
+    create_testframe_instance.open_site(url)
+    name = "Test"
+    assert_equal("Text Sent", create_testframe_instance.send_text("id", "name", name))
```

Comment:
Instead of checking the "Text Sent" message, you need to verify whether the text sent is actually present in the input box